<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>my_first</class>
 <widget class="QWidget" name="my_first">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>my_first</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>开启摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>跳转界面</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>抓拍并显示照片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>语音识别</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
